param/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 from .parameterized import logging_level # pyflakes:ignore (needed for eval)
 from .parameterized import shared_parameters # pyflakes:ignore (needed for eval)
-
+from .io import *
```

Comment:
@jbednar That makes sense to me.  I didn't realize we planned around that style of usage (putting `param` in a dif't repo directly), but I can see why people will use it.  I'll get rid of `io.py`